public/data/form-field-names.json

```diff
@@ -5,6 +5,12 @@
 		"Tab3": "Provide Your Contact Information",
 		"Tab4": ""
 	},
+	"MapDefaults": {
+		"Latitude": 39.40037792,
```

Comment:
the default lat and long are duplicated in `src/components/withFormik.jsx`. We should avoid this if possible, in the case that it would change or if we wanted to reuse this logic for another app.